optuna/distributions.py

```diff
@@ -108,6 +108,12 @@ class FloatDistribution(BaseDistribution):
     This object is instantiated by :func:`~optuna.trial.Trial.suggest_float`, and passed to
     :mod:`~optuna.samplers` in general.
 
+    .. note::
+        When ``step`` is not :obj:`None`, if the range :math:`[\\mathsf{low}, \\mathsf{high}]`
+        is not divisible by :math:`step`, :math:`\\mathsf{high}` will be replaced with
+        the maximum of :math:`k step + \\mathsf{low} < \\mathsf{high}`, where :math:`k`
```

Comment:
I didn't notice the formats for maximum are different between `IntDistribution` (`k \\times \\mathsf{step}`) and `DiscreteUniformDistribution` (`kd`). 📝  I think it would also be preferable to use `k \\times \\mathsf{step}` because `d` is replace with `step`, which is not one character variable.

Reply:
Thanks for the suggestion! Applied in ee2f4cd1c3c1ad44f69987bbf2f9a3e1379612e1.

![Screenshot from 2022-02-02 13-23-50](https://user-images.githubusercontent.com/37713008/152153418-469d6226-8cfc-4782-b2b1-4a92dc11d9b3.png)